set.mm

```diff
@@ -332247,42 +332342,43 @@ or are almost disjoint (the interiors are disjoint).  (Contributed by
       dvres.b $e |- ( ph -> B C_ S ) $.
       dvres.y $e |- ( ph -> y e. CC ) $.
       $( Lemma for ~ dvres .  (Contributed by Mario Carneiro, 8-Aug-2014.)
-         (Revised by Mario Carneiro, 28-Dec-2016.) $)
+         (Revised by Mario Carneiro, 28-Dec-2016.)  (Revised and shortened by
+         Peter Mazsa, 2-Oct-2022.) $)
```

Comment:
In order to have standard tags, write:
```
         (Revised by Mario Carneiro, 28-Dec-2016.)  Commute the consequent and
         shorten proof.  (Revised by Peter Mazsa, 2-Oct-2022.) $)
```

Reply:
done ( https://github.com/metamath/set.mm/pull/2873/commits/c72c6adbeef3a854ff3937ff8578338780a8ed2b )